namer/namer.cc

```diff
@@ -792,6 +803,23 @@ class NameInserter {
         return make_unique<ast::EmptyTree>();
     }
 
+    unique_ptr<ast::Expression> handleAssignment(core::MutableContext ctx, unique_ptr<ast::Assign> asgn) {
+        auto *send = ast::cast_tree<ast::Send>(asgn->rhs.get());
```

Comment:
This function was pulled out of `postTransformAssign` so it could be easily reused up in `handleTypeMemberDefinition`; all it does is call `fillAssign` and then, if the `Assign` node is assigning a type alias, flips the relevant flag on the symbol.